Boardfiles/discoveryl562qe/Core/Src/main.c

```diff
@@ -59,6 +59,7 @@ SD_HandleTypeDef hsd1;
 
 SPI_HandleTypeDef hspi1;
 
+TIM_HandleTypeDef htim2l

```

Comment:
Did cube ide generate this? it wont compile... there's a missing `;`

Reply:
I think that might've been there by accident. Fixed.